### ISSUE TITLE
Doublet for cleaning (Severos of Ashmunain)

### ISSUE THREAD

**Issue by MarcinKrawczuk** [BetaMasaheft/Documentation#1866]
Another one, also - it seems - not very controversial:

https://betamasaheft.eu/works/LIT1707Kebrah/main
https://betamasaheft.eu/works/LIT1299Dersan/main - that should stay, but I would add the info about Tedros Abraha's edition
https://betamasaheft.eu/works/LIT1934Mashaf/main
https://betamasaheft.eu/works/LIT1963Mashaf/main

@eu-genia @thea-m

**Comment by eu-genia** [comment]
I believe originally LIT1963Mashaf was supposed to be the Mashafa Sawiros and 1299Dersan was the ID for the Homily by Sawiros the Orthodox on St. Mary in EMML no. 1763, fols. 270r–272v. Now 1299Dersan has been used for the Mashafa Sawiros, so I guess LIT1963 must be deleted, but a new ID may be necessary for the work in EMML1763.
LIT1707Kebrah is clearly also a doublet and should be deleted.

As for 
LIT1934Mashaf 
this was originally meant to stand for Mashafa hedar 1 = Treatise on the First Four Councils - but you have created another ID for it too, which is LIT6174Severus318

(the stubs come from the EAE index, so from there it is possible to disambiguate)


Summing up, 
LIT1963Mashaf and LIT1707Kebrah should be deleted linking to 1299Dersan 
LIT1934Mashaf should be deleted linking to LIT6174Severus318

Eventually a new record should be created for the Marian Homily

**Comment by eu-genia** [comment]
Just noticed there is also LIT1729Kitaba, will delete it too

**Comment by eu-genia** [comment]
And also LIT2481Treati for LIT6174Severus318

**Comment by MarcinKrawczuk** [comment]
For EMML no. 1763, fols. 270r–272v I would suggest using LIT1665Homily. I'm OK with the rest.
